libs/ngx-nuts-and-bolts/ng-package.json

```diff
@@ -3,5 +3,6 @@
 	"dest": "../../dist/libs/ngx-nuts-and-bolts",
 	"lib": {
 		"entryFile": "src/index.ts"
-	}
+	},
+	"assets": ["./src/schematics/**/*.*"]
```

Comment:
I think this can be removed? It seems like a remnant from the initial solution before you introduced a lib where the schematics are